addon/components/sortable-item.js

```diff
@@ -169,24 +164,33 @@ export default class SortableItemComponent extends Component {
     return {
       'width': `${sortableContainer.offsetWidth}px`,
       'height': `${sortableContainer.offsetHeight}px`,
-      'background-color': PLACEHOLDER_BG_COLOR,
       'border-radius': sortableContainer.computedDOMStyles.borderRadius,
       'border-width': sortableContainer.computedDOMStyles.borderWidth,
       'margin': sortableContainer.computedDOMStyles.margin
     };
   }
 
   _onDrag(ev) {
+    if (!this.sortableContainer?.cloneNode) {
+      this._cloneDraggable();
```

Comment:
cloning while first call of on drag rather than on mousedown now. To remove some unwanted clonning.